src/huggingface_inference_toolkit/webservice_starlette.py

```diff
@@ -132,6 +144,7 @@ async def predict(request):
         routes=[
             Route(_health_route, health, methods=["GET"]),
             Route(_predict_route, predict, methods=["POST"]),
+            Route("/metrics", metrics, methods=["GET"]),
```

Comment:
Do we need it in AIP_MODE ?

Reply:
Hi here! So the short answer is no we don't, but here's more info on why we don't:

The `AIP_MODE` is the one suited specifically for Google Vertex AI (AIP stands for AI Platform on Google Cloud), and there just the `health` and `predict` endpoints are being actively used (set within Vertex AI via the `AIP_HEALTH_ROUTE` and `API_PREDICT_ROUTE` environment variables)

🤗

Reply:
Got it, I removed it 👍